app/models/conference.rb

```diff
@@ -63,10 +63,28 @@ def display_name
     acronym || id
   end
 
+  store_accessor :metadata, [ :subtitles, :logo_does_not_contain_title ]
```

Comment:
```suggestion
  store_accessor :metadata, %i[subtitles logo_does_not_contain_title]
```

Nur code style, ich habe rubocop Hinweise im Editor aktiviert, ist oft eine Hilfe.